### DISCUSSION TITLE
Como executar esse CMD pra iniciar o bot?

### DISCUSSION THREAD

**Discussion by ThiagoCamargo6** [elizandrodantas/bot-blaze-telegram#17] [General]
Pessoal, tudo bem?

Importei todas as pastas para O VS.
Porém não consigo executar aquele CMD de execução do BOT.
Como isso é feito? Alguém pode me auxiliar por favor

**Comment by DinhoNunesLC** [comment]
Eu início o bot com esse comando.

node example/example-with-dotenv.mjs

**Reply by DinhoNunesLC** [reply]
Mas recomendo usar o cmd mesmo, fora do VS, consome menos recursos da sua máquina.
